openmetadata-ui/src/main/resources/ui/src/pages/signup/index.tsx

```diff
@@ -224,24 +200,6 @@ const Signup = () => {
                       filterJoinable
                       onSelectionChange={setSelectedTeams}
                     />
-
-                    {countTeams === 0 ? (
-                      <div
-                        className="tw-notification tw-bg-info tw-mt-2 tw-justify-start tw-w-full tw-p-2"
-                        data-testid="toast">
-                        <div className="tw-font-semibold tw-flex-shrink-0">
-                          <SVGIcons
-                            alt="info"
-                            icon="info"
-                            title="Info"
-                            width="16px"
-                          />
-                        </div>
-                        <div className="tw-font-semibold tw-px-1">
-                          There is no team available.
-                        </div>
-                      </div>
-                    ) : null}
```

Comment:
Please revert this changes as we need to show this banner

Reply:
Move the badge to TeamsSelectable component.